README.md

```diff
@@ -134,30 +134,35 @@ const segments = [
+| capInnerColor               | string                                                                                                    | '#28E037'        | Cap's inner color                                                                                                                                                                                                                                                                                                                                                  |
+| capOuterColor               | string                                                                                                    | '#FFFFFF'        | Cap's outer color                                                                                                                                                                                                                                                                                                                                                  |
+| alignRangesWithSegments     | bool                                                                                                      | true             | This might be useful when using segment[].arcDegreeScale values to customize the size of individual segments. If you'd like the range display to align with the edge of each segment, pass this prop as `true`. Otherwise, range displays will be distributed evenly across the arc.                                                                               |
+| children                    | function                                                                                                  |                  | Pass a function as a child. It receives metaData with the last filled segment's data as an argument. From there you can extract data object. See example above.                                                                                                                                                                                                    |
+| dataErrorComponent          | functional component, element or null                                                                     | DataError        | This prop allows you to override the component shown below the graph when invalid props are passed (e.g., scale = NaN or scale = 0). It displays an error message to inform users that the data may be inaccurate. The library internally adjusts invalid values (e.g., changing NaN to a valid number) to prevent crashes. To hide this error, pass null instead. |
```

Comment:
I think we should include a link to `DataError` component and mention it's on by default. 
```suggestion
| dataErrorComponent          | functional component, element or null                                                                     | [DataError](./src/components/DataError.js)         | This prop allows you to override the component shown below the graph when invalid props are passed (e.g., scale = NaN or scale = 0). By default, it displays an error message to inform users that the data may be inaccurate. The library internally adjusts invalid values (e.g., changing NaN to a valid number) to prevent crashes. To hide this error, pass null instead. |
```